src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -201,161 +203,139 @@ internal async Task<List<PostgresType>> LoadBackendTypes(NpgsqlConnector conn, N
             if (timeout.IsSet)
                 commandTimeout = (int)timeout.CheckAndGetTimeLeft().TotalSeconds;
 
-            using var batch = new NpgsqlBatch(conn)
-            {
-                BatchCommands =
-                {
-                    new("SELECT version()"),
-                    new(GenerateLoadTypesQuery(SupportsRangeTypes, conn.Settings.LoadTableComposites)),
-                    new(GenerateLoadCompositeTypesQuery(conn.Settings.LoadTableComposites)),
-                }
-            };
+            var batchQuery = new StringBuilder("SELECT version();")
+                .Append(GenerateLoadTypesQuery(SupportsRangeTypes, conn.Settings.LoadTableComposites))
+                .Append(GenerateLoadCompositeTypesQuery(conn.Settings.LoadTableComposites));
```

Comment:
Just to make the query a bit more readable if anyone's looking at it (need to do the same below for the enum query):

```suggestion
            var batchQuery = new StringBuilder("SELECT version();")
                .AppendLine()
                .AppendLine(GenerateLoadTypesQuery(SupportsRangeTypes, conn.Settings.LoadTableComposites))
                .AppendLine()
                .AppendLine(GenerateLoadCompositeTypesQuery(conn.Settings.LoadTableComposites));
```

Reply:
I've changed it to contain one empty line after each query now.